src/Fractions/Fraction.Constructors.cs

```diff
@@ -50,19 +49,15 @@ public Fraction(BigInteger numerator, BigInteger denominator)
     /// </summary>
     /// <param name="numerator">integer value that will be used for the numerator. The denominator will be 1.</param>
     public Fraction(int numerator) {
-        Numerator = new BigInteger(numerator);
-        _denominator = BigInteger.One;
-        State = FractionState.IsNormalized;
+        _numerator = new BigInteger(numerator);

```

Comment:
Is there a reason why the denominator is not explicitly set to `BigInteger.One`?